src/libc/stdio.rs

```diff
@@ -152,6 +152,17 @@ fn fclose(env: &mut Environment, file_ptr: MutPtr<FILE>) -> i32 {
     }
 }
 
+fn fsetpos(env: &mut Environment, file_ptr: MutPtr<FILE>, pos: ConstPtr<i64>) -> i32 {
+    let res = i32::try_from(env.mem.read(pos)).unwrap();
+    fseek(env, file_ptr, res, SEEK_SET)
```

Comment:
It might be better to use `posix_io::lseek` so there isn't a 64-bit/32-bit conversion.

Also, you should add a type alias for `i64` here. According to the manual, the type is called `fpos_t`.

(These comments apply to `fgetpos` also.)

Reply:
Done for both